ush/forecast_postdet.sh

```diff
@@ -375,28 +375,16 @@ WW3_postdet() {
 
   # Link output files
   local wavprfx="${RUN}wave${WAV_MEMBER:-}"
-  if [[ "${waveMULTIGRID}" == ".true." ]]; then
-    ${NLN} "${COMOUT_WAVE_HISTORY}/${wavprfx}.log.mww3.${PDY}${cyc}" "log.mww3"
-    for ww3_grid in ${waveGRD}; do
-      ${NLN} "${COMOUT_WAVE_HISTORY}/${wavprfx}.log.${ww3_grid}.${PDY}${cyc}" "log.${ww3_grid}"
-    done
-  else
-    ${NLN} "${COMOUT_WAVE_HISTORY}/${wavprfx}.log.${waveGRD}.${PDY}${cyc}" "log.ww3"
-  fi
+  ${NLN} "${COMOUT_WAVE_HISTORY}/${wavprfx}.log.${waveGRD}.${PDY}${cyc}" "log.ww3"
 
   # Loop for gridded output (uses FHINC)
   local fhr vdate FHINC ww3_grid
   fhr=${FHMIN_WAV}
   fhinc=${FHOUT_WAV}
   while (( fhr <= FHMAX_WAV )); do
     vdate=$(date --utc -d "${current_cycle:0:8} ${current_cycle:8:2} + ${fhr} hours" +%Y%m%d.%H0000)
-    if [[ "${waveMULTIGRID}" == ".true." ]]; then
-      for ww3_grid in ${waveGRD} ; do
-        ${NLN} "${COMOUT_WAVE_HISTORY}/${wavprfx}.out_grd.${ww3_grid}.${vdate}" "${DATA}/${vdate}.out_grd.${ww3_grid}"
-      done
-    else
-      ${NLN} "${COMOUT_WAVE_HISTORY}/${wavprfx}.out_grd.${waveGRD}.${vdate}" "${DATA}/${vdate}.out_grd.ww3"
-    fi
+    ${NLN} "${COMOUT_WAVE_HISTORY}/${wavprfx}.out_grd.${waveGRD}.${vdate}" "${DATA}/${vdate}.out_grd.ww3"
```

Comment:
```suggestion
    ${NLN} "${COMOUT_WAVE_HISTORY}/${wavprfx}.out_grd.ww3.${vdate}" "${DATA}/${vdate}.out_grd.ww3"
```
This suggestion will have knock on effects in the wave post-processing jobs, so it might not be a good suggestion to take at this point.

Reply:
the suggestion here will ensure that there is consistency between the model produced file and the file saved in `COM`.